fastparquet/core.py

```diff
@@ -240,15 +240,25 @@ def read_data_page_v2(infile, schema_helper, se, data_header2, cmd,
                                             encoding.NumpyIO(repi), itemsize=1)
 
     max_def = schema_helper.max_definition_level(cmd.path_in_schema)
+
+    nullable = isinstance(assign.dtype, pd.core.arrays.masked.BaseMaskedDtype)
```

Comment:
This is not really considered public API, so if you need this, we should see whether we want to expose this more officially.